src/olympia/reviewers/forms.py

```diff
@@ -497,8 +514,54 @@ def clean(self):
                 raise ValidationError(
                     'Multiple policies selected with different cinder actions.'
                 )
+
+        if self.helper.actions.get(selected_action, {}).get('delayable'):
+            if 'delayed_rejection' not in self.data:
+                self.add_error(
+                    'delayed_rejection',
+                    forms.ValidationError('This field is required.'),
+                )
+            elif self.cleaned_data.get('delayed_rejection') and not self.data.get(
```

Comment:
Nit: following your other refactorings you are querying the fields from cleaned_data repeatedly and could be extracted to vars..

Reply:
a1128d64885be43823671b51a83f99ca5e212c7f